styles/variables/_typography.scss

```diff
@@ -1,7 +1,8 @@
 $font-weight-regular: 400;
 $font-weight-medium: 500;
 $font-weight-bold: 600;
-$font-weight-bolder: 900;
+$font-weight-bolder: 700;
+$font-weight-boldest: 900;
```

Comment:
I think the 600 variant should be called `semi-bold` (same as in the app), 700 should be `bold` and 900 `bolder`, so it's consistent with the [CSS naming](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight).

Reply:
2dd84afe